test/fixtures/cookbooks/test/recipes/default.rb

```diff
@@ -54,3 +54,15 @@
 file '/tmp/chef-vault-environment-secret-from-library' do
   content library_secret['type']
 end
+
+items = chef_vault('secrets')
+
+file '/tmp/chef-vault-items' do
+  content items.join(',')
+end
+
+library_items = test_chef_vault
+
+file '/tmp/chef-vault-items-from-library' do
+  content library_items.join(',')
+end
```

Comment:
You need to add matching tests to https://github.com/chef-cookbooks/chef-vault/blob/d13ff63b3b8b04d6d87715d38245ed5a746f4d03/test/integration/default/serverspec/default_spec.rb#L14